__tests__/wallet/wallet.test.ts

```diff
@@ -227,3 +236,50 @@ test('getTxBalance', async () => {
   expect(balance['token2']).toStrictEqual(-5);
 });
 
+test('generateCreateWalletAuthData should return correct auth data', async () => {
+  const requestPassword = jest.fn();
+
+  jest.spyOn(Date, 'now').mockImplementation(() => 10000);
+
+  const network = new Network('testnet');
+  const seed = 'purse orchard camera cloud piece joke hospital mechanic timber horror shoulder rebuild you decrease garlic derive rebuild random naive elbow depart okay parrot cliff';
+  const pin = '123456';
+  const wallet = new HathorWalletServiceWallet({
+    requestPassword,
+    seed,
+    network,
+    passphrase: '',
+    xpriv: null,
+    xpub: null,
+  });
+
+  const authData: CreateWalletAuthData = wallet.generateCreateWalletAuthData(pin);
+  const timestampNow = Math.floor(Date.now() / 1000); // in seconds
+
+  // these are deterministic, so we can avoid using the lib's methods to generate them
+  const xpub = 'xpub6D2LLyX98BCEkbTHsE14kgP6atagb9TR3ZBvHYQrT9yEUDYeHVBmrnnyWo3u2cADp4upagFyuu5msxtosceN1FykN22oa41o3fMEJmFG766';
+  const xprivAccountPath = 'tnpr4ugT6rZcxFWFVXBpx3YX4eS5LnUwEzGAvfbzQ9tGst2MTBjijC783X5g2LMfxPrbLJxTPHJ99nB6qDbcqehfXDVTtA33cpn49ZuA6s6WQWP';
+  const walletId = '83f704d8b24d4f9cc252b080b008280bf4b3342065f7b4baee43fd0ec7186db7';
+  const authXpub = 'xpub6AyEt1FdSvP2mXsZfJ4SLHbMugNMQVNdtkhzWoF6nQSSXcstiqEZDXd4Jg7XBscM2K9YMt2ubWXChYXMTAPS99E8Wot1tcMtyfJhhKLZLok';
+  const authXprivKey = 'tnpr4sdMdtJ7Gzh3WTc6k7bteFeLfaGc4LBPms84dQiXD8VZWb4yAY9uQFukpEaJLMpqBds8UUiamnmJWYQt97a3qGqZ5dv7yy3fWERia6CAXNh';
+  const firstAddress = 'WR1i8USJWQuaU423fwuFQbezfevmT4vFWX';
+  const xpubAddress = 'WdSD7aytFEZ5Hp8quhqu3wUCsyyGqcneMu';
+  const authXpubAddress = 'WbjNdAGBWAkCS2QVpqmacKXNy8WVXatXNM';
+
+  const privKeyAccountPath: HDPrivateKey = new HDPrivateKey(xprivAccountPath);
+  const authDerivedPrivKey: HDPrivateKey = new HDPrivateKey(authXprivKey);
+
+  const xpubkeySignature = wallet.signMessage(privKeyAccountPath, timestampNow, walletId);
+  const authXpubkeySignature = wallet.signMessage(authDerivedPrivKey, timestampNow, walletId);
+  const xpubChangeDerivation = walletUtils.xpubDeriveChild(xpub, 0);
```

Comment:
Why do you need this?

Reply:
Those 3 lines are leftovers, thanks

Removed in 7f2661426d9e44902539e6fc8733f39b8747bd99